ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -43,7 +43,7 @@ const styles = reactStyles({
   },
   formGrid: {
     display: 'grid',
-    gridTemplateColumns: '1fr 1fr 1fr 1fr 3rem 1fr',
```

Comment:
I didn't understand the 3rem in there - let me know if that was needed for some reason.

Reply:
I was seeing Disk(GB) wrap iirc.